code/src/App.js

```diff
@@ -1,9 +1,49 @@
-import React from 'react'
+import React, { useEffect, useState } from 'react'
+import { HappyThought } from './components/HappyThought'
+import { HappyForm } from 'components/HappyForm'
+
+const url = "https://technigo-thoughts.herokuapp.com/"
 
 export const App = () => {
+  const [thoughts, setThoughts] = useState([])
+  const [postedMessage, setPostedMessage] = useState("")
+
+useEffect(()=> {
+  fetch(url)
+    .then(res => res.json())
+    .then(json => setThoughts(json))
+}, [postedMessage]);
+
+const handleFormSubmit = message => {
+  fetch(url, {
+    method: "POST",
+    body: JSON.stringify({ message }),
+    headers: { "Content-Type": "application/json" }
+  })
+    .then(() => setPostedMessage(message))
+    .catch(err => console.log("error",err))
```

Comment:
Okay so about this part. You are passing `message` when you are using this function. So because you submit one message and input it here, only one will update. Hope that makes sense!